apps/src/codebridge/Console/index.tsx

```diff
@@ -11,6 +11,20 @@ import moduleStyles from './console.module.scss';
 const Console: React.FunctionComponent = () => {
   const codeOutput = useAppSelector(state => state.codebridgeConsole.output);
   const dispatch = useDispatch();
+  const levelId = useAppSelector(state => state.lab.levelProperties?.id);
+  const previousLevelId = useRef(levelId);
+  const appName = useAppSelector(state => state.lab.levelProperties?.appName);
+
+  // TODO: Update this with other apps that use the console as needed.
+  const systemMessagePrefix = appName === 'pythonlab' ? '[PYTHON LAB] ' : '';
```

Comment:
this is fine as is, but over the slightly longer term this should probably be broken out into a separate function `getSystemMessagePrefix` or something, so we have a more clear separation of concerns

Reply:
Yep makes sense, this was a quick measure to remove the hard coded python lab string.